src/api/me/autorenew/me-autorenew.v6.service.js

```diff
@@ -0,0 +1,22 @@
+angular.module('ovh-api-services').service('OvhApiMeAutorenewV6', ($resource, $cacheFactory) => {
+  const cache = $cacheFactory('OvhApiMeAutorenewV6');
+
+  const interceptor = {
+    response(response) {
+      cache.remove(response.config.url);
+      return response;
+    },
+  };
+
+  const autorenew = $resource('/me/autorenew', {}, {
+    query: { method: 'GET', cache },
```

Comment:
If it's not an array should be a `get` no ?

Reply:
No, I prefer use the `get` route for a possible future route like `/me/autorenew/:id`